apis/stacks/v1alpha1/types_test.go

```diff
@@ -29,35 +32,69 @@ var (
 )
 
 func TestStackInstallSpec_Image(t *testing.T) {
+	type want struct {
+		url string
+		err error
+	}
+
 	tests := []struct {
 		name string
 		spec StackInstallSpec
-		want string
+		want want
```

Comment:
Not a blocker, but I've been experimenting with including a human readable `reason` string that is printed when a test case fails. I find this allows succinct test/subtest names, because it removes the temptation to try to encode the full context of the test in the test name.